TRENZ.Docs.API/Controllers/FileController.cs

```diff
@@ -20,6 +20,8 @@ public FileController(ISourcesProvider sourcesProvider)
     [ResponseCache(Location = ResponseCacheLocation.Any, Duration = 60 * 60 * 24 * 7)]
     public async Task<IActionResult> Get(string location)
     {
+        // FIXME: check authorization
```

Comment:
This needs to be handled somehow. Either we apply the same logic that also applies to normal documents (meaning you need a authz file explicitly allowing groups to access a file) or we allow access from anywhere (current behavior, not preferred).

Reply:
Can't we do: if the user explicitly doesn't have access to the folder containing the file, they can't access the file either; otherwise, they can?

The typical scenario would be something like

```
Internal
- .authz
- assets
  - Screenshot.png
- Foo.md
```

Then `Foo.md` links the screenshot. If you can't access the folder containing `Foo.md`, you shouldn't be able to access the screenshot either. But if you can access the folder, I think that's fine.

Reply:
But that wouldn't prevent other users, who shouldn't see Foo.md, but other files in this folder from viewing the screenshot. One could argue that this is irrelevant because a member of a different group wouldn't know about the path of the screenshot. I'm just not a fan of "security by obscurity".

Reply:
It would be security by obscurity if *anyone* could access the file by knowing the URL. But only those who have access to the folder above it would be able to.

Reply:
Internal
- .authz           | allows group 1 to access Foo.md and Bar.md, allows group 2 to access Bar.md
- Foo.md           | embeds assets/sensitive.png
- Bar.md           | embeds assets/screenshot.png
- assets           | every group having access to Internal can see everything inside here
  - screenshot.png | every group can see this file
  - sensitive.png  | oh-oh! group 2 can access sensitive.png even though only group 1 was intended

Reply:
I get that, but the only way to resolve it would be to have individual auth information for asset files. I don't think we want that.